openedx/stanford/lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -218,8 +220,8 @@ def update_task_progress():
         header, datarows = csv_fn(course_id)
         rows = [header] + [row for row in datarows]
     # Update progress to failed regardless of error type
-    # pylint: disable=bare-except
```

Comment:
why are removing this pylint disabling?

Reply:
Take another look at _which_ pylint error is removed...

Reply:
oh DOH, ok nvmd